packages/0x.js/src/contract_wrappers/ether_token_wrapper.ts

```diff
@@ -27,10 +27,13 @@ export class EtherTokenWrapper extends ContractWrapper {
      * to the depositor address. These wrapped ETH tokens can be used in 0x trades and are redeemable for 1-to-1
      * for ETH.
      * @param   amountInWei      Amount of ETH in Wei the caller wishes to deposit.
-     * @param   depositor   The hex encoded user Ethereum address that would like to make the deposit.
+     * @param   depositor        The hex encoded user Ethereum address that would like to make the deposit.
+     * @param   txOpts           Transaction parameters.
```

Comment:
`Optional transaction parameters`

Reply:
We don't have the convention to show optionality as a comment cause it can easily seen from the type.
https://0xproject.com/docs/0xjs#awaitTransactionMinedAsync